esmvaltool/diag_scripts/shared/plot/_plot.py

```diff
@@ -9,8 +9,58 @@
 logger = logging.getLogger(__name__)
 
 
-def get_path_to_mpl_style(style_file):
+def _process_axes_functions(axes, axes_functions):
+    """Process axes functions of the form `axes.functions(*args, **kwargs)."""
+    if axes_functions is None:
+        return None
+    output = None
+    for (func, attr) in axes_functions.items():
+        axes_function = getattr(axes, func)
+
+        # Simple functions (argument directly given)
+        if not isinstance(attr, dict):
+            try:
+                out = axes_function(*attr)
+            except TypeError:
+                out = axes_function(attr)
+
+        # More complicated functions (args and kwargs given)
+        else:
+            args = attr.get('args', [])
+            kwargs = attr.get('kwargs', {})
+
+            # Process 'transform' kwargs
+            if 'transform' in kwargs:
+                kwargs['transform'] = getattr(axes, kwargs['transform'])
+            out = axes_function(*args, **kwargs)
+
+        # Return legend if possible
+        if func == 'legend':
+            output = out
+    return output
+
+
+def _check_size_of_parameters(*args):
+    """Check if the size of (array-like) args is identical."""
+    if len(args) < 2:
+        logger.warning("Less than two arguments given, comparing not possible")
+        return None
+    arg_0 = args[0]
+    for arg in args:
+        try:
+            if len(arg_0) != len(arg):
+                raise ValueError("Invalid input: array-like parameters need "
+                                 "to have the same size")
+        except TypeError:
+            raise TypeError("Invalid input: some parameters are not "
+                            "array-like")
+    return None
```

Comment:
There is no need to write `return None`, any function that does not have a return statement returns `None`

Reply:
Yeah, I know that, but prospector wants me to do that!

```
esmvaltool/diag_scripts/shared/plot/_plot.py
  Line: 43
    pylint: inconsistent-return-statements / Either all return statements in a function should return an expression, or none of them should.
```